README.md

```diff
@@ -58,4 +110,159 @@ are empty or missing, dashboard is effectively disabled.
         "access_dashboard_groups": ["mcka_role_client_admin", "mcka_role_internal_admin"],
         "ta_roles": ["assistant"]
       }
-    }
\ No newline at end of file
+    }
+    
+    
+### Enabling Group Project XBlock v2 in course
+
+To enable the use of Group Project XBlock v2 in the course:
+
+1. Open the course in Studio.
+2. In the top menu choose Settings -> Advances Settings
+3. Add `gp-v2-project` to "Advanced Module List"
+4. Optionally, add `ooyala-player` to "Advanced Module List" - this player is used for video resources
+
+As a result, `Advanced Module List` option should look like that (provided there are no other advanced modules are 
+enabled for course): 
+![Advanced Module List Image](/docs/images/advanced_module_list.png)
+
+
+### (Optional) Notifications integration
+
+Group Project XBlock v2 sends a number of notifications via edx-notifications app. In order to make those notifications
+appear correctly, make sure `NOTIFICATION_CLICK_LINK_URL_MAPS` Django setting contains the following record:
+
+    'open-edx.xblock.group-project-v2.*': '/courses/{course_id}/group_work?activate_block_id={location}'
+    
+If you're using edx-solutions/edx-platform fork, appropriate value is already set. Otherwise, note that 
+`NOTIFICATION_CLICK_LINK_URL_MAPS` are read from `lms.env.json`, so they should be modified there.    
+
+Example:
+
+```
+NOTIFICATION_CLICK_LINK_URL_MAPS = {
+    'open-edx.studio.announcements.*': '/courses/{course_id}/announcements',
+    'open-edx.lms.leaderboard.*': '/courses/{course_id}/cohort',
+    'open-edx.lms.discussions.*': '/courses/{course_id}/discussion/{commentable_id}/threads/{thread_id}',
+    'open-edx.xblock.group-project.*': '/courses/{course_id}/group_work?seqid={activity_location}',
+    'open-edx.xblock.group-project-v2.*': '/courses/{course_id}/group_work?activate_block_id={location}',
+}
+```  
+
+
+# Configuration
+
+In order to have a working Group Project, one need three components:
+1. Author the XBlock in Studio
+2. Set up environment configuration variables (covered earlier)
+3. Configure Group Project in Apros (3rd party LMS) 
```

Comment:
@e-kolpakov "one need three components" ⟶ "you'll need to complete three steps"?

Also, it might make sense to swap points 1 and 2.